raiden_contracts/deploy/contract_deployer.py

```diff
@@ -170,7 +170,7 @@ def deploy_raiden_contracts(
 
         deployed_contracts: DeployedContracts = {
             'contracts_version': self.contract_version_string(),
-            'chain_id': int(self.web3.version.network),
+            'chain_id': int(self.web3.eth.protocolVersion),
```

Comment:
I think these replacements are wrong: ` web3.eth.protocolVersion` is the protocol version (which is something in the 60s right now) and `web3.version.network` is the chain or network id. You can access it with `web3.net.version`.

From a tester chain:
```
(Pdb++) web3.net.version
'1'
(Pdb++) web3.eth.protocolVersion
'99'
(Pdb++) web3.version.network
'1'
```

Reply:
Thanks for checking this.